bmorph/evaluation/plotting.py

```diff
@@ -16,6 +16,15 @@
 
 from statsmodels.distributions.empirical_distribution import ECDF
 
+#*****************************************************************************************
+# Plotting Helper Functions:
+#      custom_legend
+#      calc_water_year
+#      find_index_hyear
+#      determin_row_col
+#      log10_1p
+#      scatter_series_axes
+#*****************************************************************************************
```

Comment:
I like the idea of having this, but it would be better inside of a docstring so that we can build an automated documentation website from it eventually. Basically just:

```
> ``` 
> # Plotting Helper Functions:
>      - custom_legend
>      - calc_water_year
>      - find_index_hyear
>      - determin_row_col
>      - log10_1p
>      - scatter_series_axes
> ```
```